configure

```diff
@@ -173,6 +173,8 @@ boost_lib_dir=""
 boost_inc_dir=""
 eigen_inc_dir=""
 BOOSTFLAGS=""
+HIGGSTOOLSFLAGS="-Imodel_specific/SM"
+HIGGSTOOLSLIBS="model_specific/SM/libmodel_specific_SM.a"
```

Comment:
Please use `${STATIC_LIB_EXT}` instead of `.a`, i.e.
~~~.sh
HIGGSTOOLSLIBS="model_specific/SM/libmodel_specific_SM.${STATIC_LIB_EXT}"
~~~
(This may also apply to your other changes below.)

Reply:
Done in de968f92a015185a0b76f2e9d39007b23685a3d5.